src/foundry/client/pixi/layers/base/interaction-layer.d.ts

```diff
@@ -0,0 +1,135 @@
+export {};
+
+declare global {
+  /**
+   * A subclass of CanvasLayer which provides support for user interaction with its contained objects.
+   */
+  class InteractionLayer extends CanvasLayer {
+    /**
+     * Is this layer currently active
+     */
+    get active(): boolean;
+
+    /**
+     * @privateRemarks Override not in foundry docs but implicit from layerOptions
+     */
+    override options: InteractionLayer.LayerOptions;
+
+    /**
+     * Customize behaviors of this CanvasLayer by modifying some behaviors at a class level.
+     */
+    static get layerOptions(): InteractionLayer.LayerOptions;
+
+    /**
+     * Activate the InteractionLayer, deactivating other layers and marking this layer's children as interactive.
+     * @param options - Options which configure layer activation
+     * @returns - The layer instance, now activated
+     */
+    activate(options?: {
```

Comment:
Probably apply `InexactPartial`. I'll be noting this a lot, maybe you have a reason to avoid it?